plugins/Diagnostics/Diagnostic/ModSecurityCheck.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+namespace Piwik\Plugins\Diagnostics\Diagnostic;
+
+use Piwik\Config;
+use Piwik\Filechecks;
+use Piwik\Http;
+use Piwik\SettingsServer;
+use Piwik\Translation\Translator;
+
+/**
+ * Check that Piwik's mod security is off.
+ */
+class ModSecurityCheck implements Diagnostic
+{
+    /**
+     * @var Translator
+     */
+    private $translator;
+
+    public function __construct(Translator $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    public function execute()
+    {
+        $label = $this->translator->translate('Installation_SystemCheckModSecurity');
+
+        $modSecurity = in_array('mod_rewrite', get_loaded_extensions()) !== false;
```

Comment:
This doesn't look right. Shouldn't that be `mod_security` or something similar? Haven't checked that, but is mod_security even available in the loaded extensions? Or wouldn't we need to look for apache extensions here?